### ISSUE TITLE
Inserting more than 2048 documents one after another fails (promise not resolved)

### ISSUE THREAD

**Issue by gyszalai** [apache/couchdb-nano#234]
## Expected Behavior
When I try inserting more than 2048 documents one after another, I expect success

## Current Behavior
Inserting the 2048th document fails, i.e. the promise is never resolved.

## Possible Solution
No clue

## Steps to Reproduce (for bugs)
I created a sample project for repoducing the bug: https://github.com/gyszalai/nano9_bug
This project starts couchdb 2.3.1 and 3.1.1 with docker-compose and tries to insert 3000-3000 document into both.

### Steps to reproduce:
* clone the project
* npm install
* npm start
* it should be successful
* npm i nano@9.0.0
* npm start
* it will insert 2047 documents in couchdb 2.3.1 and then stops inserting more

## Context
We have an API that accepts messages that it tries to put into couchdb

## Your Environment
* Version used: 9.0.0
* Browser Name and version: -
* Operating System and version (desktop or mobile): macOS 10.15.7, node 14.15.0
* Link to your project: -

**Comment by glynnbird** [comment]
Thanks for reporting this. 

Simplifying the code example to this:

```
const couchUrl = 'http://admin:admin@localhost:5984'
const nano = require('nano')(couchUrl)
const dbName = 'mydb'
const db = nano.use(dbName)
async function run () {
  await nano.db.destroy(dbName)
  await nano.db.create(dbName)
  for (let i = 0; i <2100 ; i++) {
    console.log('idx: '+ i)
    await db.insert({ _id: 'key_' + i, someProp: 'test_' + i })
  }
}

run()
```

**Comment by glynnbird** [comment]
Thanks for this report. It has been fixed in the 9.0.1 release 💯 

As an aside, if you want to create hundreds of documents, then consider using the [bulk](https://github.com/apache/couchdb-nano#dbbulkdocs-params-callback) function to insert them in bulk.

**Comment by gyszalai** [comment]
> Thanks for this report. It has been fixed in the 9.0.1 release 💯
> 
> As an aside, if you want to create hundreds of documents, then consider using the [bulk](https://github.com/apache/couchdb-nano#dbbulkdocs-params-callback) function to insert them in bulk.

WOW! Thanks for fixing it this quickly! 

I know that I can use the bulk function, but we have an API that receives many independent requests with only one document per request, so saving them with the bulk function is not possible.
